models.go

```diff
@@ -19,8 +21,22 @@ type Team struct {
 
 // Account represents the configuration for an assumable role.
 type Account struct {
-	Name    string `json:"name"`
-	RoleArn string `json:"roleArn"`
+	Name     string `json:"name"`
+	RoleArn  string `json:"roleArn"`
+	Duration int    `json:"duration"`
+}
+
+func (a *Account) UnmarshalJSON(b []byte) error {
+	type accountType Account
+	if err := json.Unmarshal(b, (*accountType)(a)); err != nil {
+		return err
+	}
+	if a.Duration == 0 {
+		a.Duration = 3600
+	} else if a.Duration < 3600 || a.Duration > 43200 {
+		return fmt.Errorf("duration should be between 3600 and 43200 seconds but is: %d", a.Duration)
```

Comment:
In general I have left validation to the STS API's and log errors, so one account can be misconfigured without breaking other accounts. In short, I'd rather not implement a custom `UnmarshalJSON`.